src/command/dev/next/router/config/mod.rs

```diff
@@ -54,6 +54,19 @@ impl RouterAddress {
     }
 }
 
+impl Display for RouterAddress {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let host = self
+            .host
+            .to_string()
+            .replace("127.0.0.1", "localhost")
+            .replace("0.0.0.0", "localhost")
+            .replace("[::]", "localhost")
+            .replace("[::1]", "localhost");
+        write!(f, "http://{}:{}", host, self.port)
+    }
+}
+
```

Comment:
This logic is directly copied over from the old version of the log line that says "your supergraph is ready" etc.